src/java-runtime/index.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import findJavaHome = require("find-java-home");
+
+export async function validateJavaRuntime(): Promise<boolean> {
```

Comment:
For VSCode Java, it can directly use the java.home user setting. If user specified in the VSCode, it doesn't required additional JDK. The find home is based on JAVA_HOME value without knowledg of VSCode itself.